stable/prometheus/values.yaml

```diff
@@ -863,7 +851,9 @@ pushgateway:
 
   ## Security context to be added to push-gateway pods
   ##
-  securityContext: {}
+  securityContext: 
+    runAsUser: 65534
+    runAsNonRoot: true
```

Comment:
Why not add fsGroup and runAsGroup to all the securityContexts?

Reply:
because those components doesn't create any files.
i added fsGroup and runAsGroup only to components, which write to files during runtime.